### DISCUSSION TITLE
using a ticker.yaml does not explain what will be the command to run it

### DISCUSSION THREAD

**Discussion by ghost** [achannarasappa/ticker#45] [Q&A]
Hi,

thank you for this great useful code I will daily use but ...
Even if I used the recommended yaml file I can not display this content by using even "ticker" or "ticker -w".
Then I read back twice the explanation but did not found how to run it  correctly. I even moved the yaml file from /usr/local/bin to /home or /home/<my account> and did not get it workable.

Please tell me how to use it properly with yaml.

Have a nice day and take care.

**Comment by aqsmith08** [comment]
Hey - I had this issue too but I was able to figure it out. After installing ticker and creating the `ticker.yaml` file, here's the command I ran in the command line: `ticker --config ticker.yaml`. I made sure I was in the directory where the `ticker.yaml` file existed. Give that a try!
